src/main/java/org/jfrog/hudson/pipeline/common/types/deployers/MavenDeployer.java

```diff
@@ -102,4 +114,44 @@ private static MavenDeployer createEmptyDeployer() {
         dummy.setThreads(1);
         return dummy;
     }
+
+    public static void addDeployedMavenArtifactsActionFromModules(Run build, String artifactoryUrl, List<Module> modules) {
+        for (Module module : modules) {
+            if (module.getArtifacts() != null) {
+                addDeployedMavenArtifactsAction(build, artifactoryUrl, module.getArtifacts());
+            }
+        }
+    }
+
+    public static void addDeployedMavenArtifactsActionFromDetails(Run build, String artifactoryUrl, Map<String, Set<DeployDetails>> deployableArtifactsByModule) {
```

Comment:
Let's add a description, explaining what this method does.
addDeployedMavenArtifactsActionFromDetails --> addDeployedArtifactsActionFromDetails